src/Uno.UWP/Uno.csproj

```diff
@@ -52,11 +52,13 @@
 		<PackageReference Include="Xamarin.AndroidX.Legacy.Support.v4" />
 		<PackageReference Include="Xamarin.AndroidX.AppCompat" />
 		<PackageReference Include="Xamarin.AndroidX.Fragment" />
+		<PackageReference Include="Xamarin.AndroidX.Browser"  Version="1.0.0" />
```

Comment:
Taking a dependency here [requires the nuspec](https://github.com/unoplatform/uno/blob/6240b58a4d94e43bba3567763866a15de62041aa/build/Uno.WinUI.nuspec#L17) to include it as well.

Reply:
done